_posts/2018-06-21-ice-public-interest.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "In Defense of Sam Lavigne, and why publishing LinkedIn data on ICE employees is in the public interest"
+image: "images/blogimages/Ice_smart_r_80.png"
+author: camille_fassett and jennifer_helsby
+teaser: "Sharing information about government agents related to the execution of their public duties for the purposes of government accountability and transparency should never be considered doxxing or harassment."
+categories:
+tags:
+  - ice
+  - accountability
+  - data
+  - public interest
+---
+
+Motivated by the Trump administration’s [policy](https://www.washingtonpost.com/news/fact-checker/wp/2018/06/19/the-facts-about-trumps-policy-of-separating-families-at-the-border/) of detaining and separating migrant children from their families at the border, artist and developer Sam Lavigne harvested profiles that list Immigration and Customs Enforcement (ICE) as an employer on social network LinkedIn.
+
+He made the dataset of all 1595 responsive profiles, which included names, titles, photographs and general locations, available to download on GitHub. The repository also included the Python script that he used to scrape LinkedIn, along with basic instructions on how to replicate his work.
+
+“While I don’t have a precise idea of what should be done with this data set, I leave it here with the hope that researchers, journalists and activists will find it useful,” Lavigne [wrote](https://web.archive.org/web/20180619140914/https://medium.com/@samlavigne/downloading-the-profiles-of-everyone-on-linkedin-who-works-for-ice-c4e0ff6b065e) in a Medium post about the profiles he scraped.
+
+But at the URL where the dataset was hosted is now just a page that reads: “This repository has been disabled. Access to this repository has been disabled by GitHub staff.” And in place of Lavigne’s explainer on Medium is simply, “This page is unavailable.”
+
+GitHub and Medium aren’t the only entities that scrubbed Lavigne’s work from their platforms. Twitter also suspended @iceHRgov, an account that tweeted profiles from the dataset.
```

Comment:
I think adding backticks around `@iceHRgov` would be good here

Reply:
added in cff6dc9